### ISSUE TITLE
Table optional Bold First.

### ISSUE THREAD

**Issue by TheChileanKink** [DavBfr/dart_pdf#263]
**Is your feature request related to a problem? Please describe.**
I'd like to chose whether or not I want to bold the first Row of a Table.

**Describe the solution you'd like**

```dart
factory Table.fromTextArray({
    @required Context context,
    @required List<List<String>> data,
    bool boldFirst=true,
    EdgeInsets margin = const EdgeInsets.all(5),
  }) {
    final List<TableRow> rows = <TableRow>[];
    for (List<String> row in data) {
      final List<Widget> tableRow = <Widget>[];
      if (row == data.first && boldFirst) {
        for (String cell in row) {
          tableRow.add(Container(
              alignment: Alignment.center,
              margin: margin,
              child: Text(cell, style: Theme.of(context).tableHeader)));
        }
      } else {
...
```

**Describe alternatives you've considered**
Adding an Empty Row which leads to a small gap on the Table.
Adding an empty List on the beginning of the List<List<String>> data. The for loop doesn't run since there is nothing to bold.

**Comment by DavBfr** [comment]
Do you want to do it and provide a PR?

**Comment by DavBfr** [comment]
This has been added to the master branch.

**Comment by DavBfr** [comment]
Published in pdf 1.8.0
Don't hesitate to [Buy Me A Coffee](https://www.buymeacoffee.com/JORBmbw9h).
